### PR TITLE
loading api data into div

### PR DESCRIPTION
**Pull request by jcorteza**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,10 +2,32 @@
...
-    <meta name="viewport" content="width=device-width, intitial-scale=1.0"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Onboarding Web Page</title>
 </head>
 <body>
     <h1>Hello World</h1>
+    <button id="apiButton" type="button" >Get Twitter Timeline</button>
+    <div id="apiDataContainer"></div>
+
+    <script type="application/javascript">
```
Comment (kev-chan) [inline src/index.html]: let's put this in a separate js file

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: src/js/functional.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let request = new XMLHttpRequest();
+    const btn = document.getElementById("apiButton");
+    
+    btn.addEventListener("click", (e) => {
+        
+        e.preventDefault();
+        request.addEventListener("load", function() {
+            const apiDataContainer = document.getElementById("apiDataContainer");
```
Comment (kev-chan) [inline src/js/functional.js]: what if the call fails?

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: src/js/main.js [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
 const handleRequest = (request, response) => {
 
-    fs.readFile("./src/index.html", "utf8", (err, page) => {
+    let filePath = `.${request.url}`;
+    if(filePath == "./") {
+        filePath = "index.html";
+    }
 
-        if(err) throw err;
+    let extname = path.extname(filePath);
```
Comment (kev-chan) [inline src/js/main.js:13]: nice

File: src/index.html [hunk 4]
```diff
@@ -2,10 +2,14 @@
...
-    <meta name="viewport" content="width=device-width, intitial-scale=1.0"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Onboarding Web Page</title>
 </head>
 <body>
     <h1>Hello World</h1>
+    <button id="apiButton" type="button" >Get Twitter Timeline</button>
+    <div id="apiDataContainer"></div>
+
+    <script type="application/javascript" src="./js/functional.js"></script>
```
Comment (kev-chan) [inline src/index.html]: put this in the head element

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 4 above)
Comment (jcorteza) [reply src/index.html]: I put it in the body, as a page load performance thing

**Pull request merged**